pages/establishment/dashboard/views/index.jsx

```diff
@@ -30,33 +30,49 @@ const Index = ({
     pelh,
     nprc
   },
-  allowedActions
-}) => (
-  <Fragment>
-    <Header title={name} />
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-two-thirds">
-        <PanelList
-          panels={links.filter(link => allowedActions.includes(link.permissions)).map(link => <DashboardLink key={link.path} { ...link } />)}
-        />
+  allowedActions,
+  profile: {
+    asruAdmin
+  }
```

Comment:
Assuming you take my suggestion below, then `asruAdmin` would be a top-level prop.
```suggestion
  allowedActions,
  asruAdmin
```

Reply:
done